sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -92,19 +92,19 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake { nam
 
   // PathClient models:
 
-  struct DeleteDataLakePathResult
+  struct DeletePathResult
   {
     bool Deleted = true;
     std::string RequestId;
   };
 
-  using AcquireDataLakeLeaseResult = Blobs::Models::AcquireBlobLeaseResult;
-  using RenewDataLakeLeaseResult = Blobs::Models::RenewBlobLeaseResult;
-  using ReleaseDataLakeLeaseResult = Blobs::Models::ReleaseBlobLeaseResult;
-  using ChangeDataLakeLeaseResult = Blobs::Models::ChangeBlobLeaseResult;
-  using BreakDataLakeLeaseResult = Blobs::Models::BreakBlobLeaseResult;
+  using AcquireLeaseResult = Blobs::Models::AcquireLeaseResult;
+  using RenewLeaseResult = Blobs::Models::RenewLeaseResult;
+  using ReleaseLeaseResult = Blobs::Models::ReleaseLeaseResult;
+  using ChangeLeaseResult = Blobs::Models::ChangeLeaseResult;
+  using BreakLeaseResult = Blobs::Models::BreakLeaseResult;
   using RehydratePriority = Blobs::Models::RehydratePriority;
```

Comment:
Interesting that this one didn't need to be renamed. I wonder if it was missed in some previous update which added `DataLake` to others.

Reply:
Some of this kind of stuff is not consistent. Like `LeaseStatus` in datalake service, while `BlobLeaseStatus` in blob service

Reply:
I don't know if I should remove those `Blob` prefixes.

Because in the email you said `Leave client names and **names of resource models** as-is.`

I don't know what should be considered **resource models**

Reply:
I don't consider the lease status as a resource.

We are somewhat inconsistent in the name already:
https://github.com/Azure/azure-sdk-for-cpp/blob/a5bc8fe0efa4534a45692c2787642ba1f073a6da/sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp#L261
https://github.com/Azure/azure-sdk-for-cpp/blob/a5bc8fe0efa4534a45692c2787642ba1f073a6da/sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp#L187
https://github.com/Azure/azure-sdk-for-cpp/blob/a5bc8fe0efa4534a45692c2787642ba1f073a6da/sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp#L230

We should aim for local consistency here and rename `BlobLeaseStatus` to `LeaseStatusType` (and use the namespaces to differentiate the whole typename). Is there a reason for them to be named differently for some reason?

Reply:
> Is there a reason for them to be named differently for some reason?

There's no reason. Just Because I developed Blob generator, while @katmsft did datalake/file generator. So this kind of changes are totally understandable.

Reply:
> We should aim for local consistency here and rename BlobLeaseStatus to LeaseStatusType (and use the namespaces to differentiate the whole typename). 

Then I'll remove `Blob` prefix for all these stuff

Reply:
If the service itself has this type of inconsistency, then see what other languages are doing. Do we go with the service as the "source of truth" in other places?

Looks like .NET calls it LeaseStatus in all places.

Reply:
> Then I'll remove `Blob` prefix for all these stuff

What else had this issue that you were thinking to update?

Reply:
Nothing else I can think of

Reply:
All removed

Reply:
Btw, about the definition of resource models, typically its the return types that are named just `<Model>` and not `<Operation>Result`. You'd want to keep those types unique, just like the clients.

Reply:
> Btw, about the definition of resource models, typically its the return types that are named just <Model> and not <Operation>Result. You'd want to keep those types unique, just like the clients.

Agree. Currently all of them are unique and I will keep them unique.